community/index.md

```diff
@@ -36,6 +36,7 @@ Third party companies also provide commercial support and services for Druid, in
 
 * [Cloudera](https://www.cloudera.com/)
 * [Datumo](https://datumo.io/)
+* [Deep.BI](/https://www.deep.bi/solutions/apache-druid)
```

Comment:
The leading slash here is going to make the link go to the wrong place, could you please fix it?

Reply:
@gianm thanks so much for catching that! Fixing it now.